api/tests/core/offerers/test_tasks.py

```diff
@@ -54,7 +55,35 @@ def test_tag_inactive_offerer(self, mock_get_siren, client, siren_caduc_tag):
         assert offerer.tags == [siren_caduc_tag]
 
     @patch("pcapi.connectors.sirene.get_siren")
-    def test_do_not_tag_inactive_offerer(self, mock_get_siren, client, siren_caduc_tag):
+    def test_offerer_with_pending_status_should_be_rejected_if_tag_inactif(
+        self, mock_get_siren, client, siren_caduc_tag
+    ):
+        offerer = offerers_factories.OffererFactory(
+            validationStatus=ValidationStatus.PENDING,
+        )
+
+        mock_get_siren.return_value = sirene.SirenInfo(
+            siren=offerer.siren,
+            name=offerer.name,
+            head_office_siret=offerer.siren + "00001",
+            ape_code="6201Z",
+            legal_category_code="1000",
+            address=None,
+            active=False,
+        )
+
+        response = client.post(
+            f"{settings.API_URL}/cloud-tasks/offerers/check_offerer_is_active",
+            json={"siren": offerer.siren, "tag_when_inactive": True},
+            headers={AUTHORIZATION_HEADER_KEY: AUTHORIZATION_HEADER_VALUE},
+        )
+
+        assert response.status_code == 204
+        assert offerer.validationStatus == ValidationStatus.REJECTED
+        assert offerer.tags == [siren_caduc_tag]
+
+    @patch("pcapi.connectors.sirene.get_siren")
+    def test_do_not_tag_the_offerer_if_tag_is_actif(self, mock_get_siren, client, siren_caduc_tag):
```

Comment:
Pourquoi changer le nom pour quelque chose mi-anglais mi-français qui ne correspond pas au cas ?
```suggestion
    def test_do_not_tag_inactive_offerer(self, mock_get_siren, client, siren_caduc_tag):
```